duckduckgo/iOS
# Autofill Ship Review updates

Task/Issue URL: https://app.asana.com/0/38424471409662/1204265086393897/f
Tech Design URL:
CC:

**Description**:
Minor updates from Ship Review feedback



**Steps to test this PR**:
Note: Updated translations are in progress

**From the Autofill prompt the first / top button option is always solid blue (regardless of subdomain) https://app.asana.com/0/1203822806345703/1204391947652310/f**

_Screenshot of design changes below_
1. Add a login which is an exact match for the website e.g. `fill.dev` 
2. Confirm the autofill prompt displays this option as a solid blue button
3. Modify this login so it's url is a partial match to the website e.g. `sub.fill.dev`
4. Confirm the autofill prompt displays this option as a solid blue button
5. Add a mix of matching and partial matching logins for `fill.dev` and confirm the other button options are white with a blue border (as before)

  ![image](https://user-images.githubusercontent.com/91189922/233115680-74d11056-c740-4b80-8ebd-e7f81ea180b2.png)



**Design updates https://app.asana.com/0/0/1204272093186570/f**

Note: These have already been checked as part of the design review and signed off 
1. Confirm the Save login prompt and the Autofill suggestion prompt have the same blue buttons ([asana task](https://app.asana.com/0/0/1204354327855766/f))
2. Confirm that when there are **no** logins saved there is no `Edit`button in the navigation bar (regardless of Autofill switch state) ([asana task](https://app.asana.com/0/38424471409662/1204371654436577/f))
3. Confirm that when there **are** logins saved the `Edit`button is in the navigation bar (regardless of Autofill switch state)
4. Confirm there is now more vertical spacing between sections on the Autofill list screen ([asana task](https://app.asana.com/0/0/1204371654436589/f))
5. When a user has a login saved with only a password and no username, confirm the button text is now `Password for <URL>` ([asana task](https://app.asana.com/0/38424471409662/1204374791583417/f))


**Copy updates https://app.asana.com/0/0/1204272090501059/1204407495626943/f** 

Note: These have already been checked as part of the copy review and signed off 
1. In the Logins screen with no Logins saved, the title should be `No Logins saved yet`
2. The prompt to save a new login should be `Logins are stored securely on this device only, and can be managed from the Logins menu in Settings.`
3. In the Logins menu search bar, the placeholder text should be `Search Logins`.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
